result/rsvp.yaml

```diff
@@ -0,0 +1,192 @@
+            The number of Resv messages sent by this RSVP router.
+          type: integer
+        resvs_rx:
+          description: >-
+            The number of Resv messages received by this RSVP router.
+          type: integer
+        path_tears_tx:
+          description: >-
+            The number of  Path Tear messages sent by this RSVP router.
+          type: integer
+        path_tears_rx:
+          description: >-
+            The number of Path Tear messages received by this RSVP router. 
+          type: integer
+        resv_tears_tx:
+          description: >-
+            The number of  Resv Tear messages sent by this RSVP router.
+          type: integer
+        resv_tears_rx:
+          description: >-
+            The number of Resv Tear messages received by this RSVP router. 
+          type: integer
+        path_errors_tx:
+          description: >-
+            The number of Path Error messages sent by this RSVP router.
+          type: integer
+        path_errors_rx:
+          description: >-
+            The number of Path Error messages received by this RSVP router. 
+          type: integer
+        resv_errors_tx:
+          description: >-
+            The number of Resv Error messages sent by this RSVP router.
+          type: integer
+        resv_errors_rx:
+          description: >-
+            The number of Resv Error messages received by this RSVP router. 
+          type: integer
+        resv_conf_tx:
+          description: >-
+            The number of ResvConf messages sent by this RSVP router.
+          type: integer
+        resv_conf_rx:
+          description: >-
+            The number of ResvConf messages received by this RSVP router. 
+          type: integer
+        hellos_tx:
+          description: >-
+            The number of Hello messages sent by this RSVP router.
+          type: integer
+        hellos_rx:
+          description: >-
+            The number of Hello messages received by this RSVP router. 
+          type: integer
+        acks_tx:
+          description: >-
+            The number of Ack messages sent by this RSVP router.
+          type: integer
+        acks_rx:
+          description: >-
+            The number of Ack messages received by this RSVP router. 
+          type: integer
+        nacks_tx:
+          description: >-
+            The number of Nack messages sent by this RSVP router.
+          type: integer
+        nacks_rx:
+          description: >-
+            The number of Nack messages received by this RSVP router. 
+          type: integer
+        srefresh_tx:
+          description: >-
+            The number of SRefresh messages sent by this RSVP router.
+          type: integer
+        srefresh_rx:
+          description: >-
+            The number of SRefresh messages received by this RSVP router. 
+          type: integer
+        bundle_tx:
+          description: >-
+            The number of SRefresh messages sent by this RSVP router.
```

Comment:
Same description as that for 'srefresh_tx'.  Is that correct?

Reply:
This was a mistake. Changed to Bundle messages.